google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -249,7 +249,7 @@ def parse_insert(insert_sql, params):
         it produces:
             {
                 'sql_params_list': [
-                    ('INSERT INTO T (f1, f2) VALUES (%s, LOWER(%s))', ('a', 'b',))
+                    ('INSERT INTO T (f1, f2) VALUES (%s, LOWER(%s))', ('a', 'b',)),
```

Comment:
A comma is missing here - it's a `list`, so inner tuples must be separated with a comma.